Doc/library/functools.rst

```diff
@@ -137,7 +137,7 @@ The :mod:`functools` module defines the following functions:
                lru_cache(maxsize=128, typed=False)
 
    Decorator to wrap a function with a memoizing callable that saves up to the
-   *maxsize* most recent calls.  It can save time when an expensive or I/O bound
+   *maxsize* most recent calls.  It can save time when an expensive or I/O-bound
```

Comment:
In this case, `I/O bound` precedes `function`, so it should be hyphenated.

["Use a hyphen in a compound modifier when the modifier comes before the word it’s modifying."](https://www.grammarly.com/blog/hyphen/)